src/coreComponents/mesh/generators/CellBlockABC.hpp

```diff
@@ -76,6 +76,12 @@ class CellBlockABC : public dataRepository::Group
    */
   virtual localIndex numFacesPerElement() const = 0;
 
+  /**
+   * @brief Get the maximum number of nodes comprising a face.
+   * @return maximum number of nodes comprising a face.
+   */
+  virtual localIndex maxNodesPerFace() const = 0;
+
```

Comment:
Without diving into the details, I do not think it's required to expose a new function `maxNodesPerFace` into the `CellBlockABC` contract.
This `maxNodesPerFace` information looks like it's a computational information to manage a tricky corner case.
But once the issue's solved, it's not required anymore, is it? (From your PR, it appears that people manipulating `CellBlockABC` abstractions never call `CellBlockABC::maxNodesPerFace`: only the `CellBlock` implementation gets called. Hence `CellBlock::maxNodesPerFace` should not be exposed too broadly.).

I think that if you remove this `virtual` member function, the code will still run fine (expect `CellBlock::maxNodesPerFace` which cannot be defined `override` obviously).

Reply:
Yes good idea.